exporter/kafkaexporter/README.md

```diff
@@ -61,6 +61,14 @@ The following settings can be optionally configured:
   User should calculate this as `num_seconds * requests_per_second` where:
     - `num_seconds` is the number of seconds to buffer in case of a backend outage
     - `requests_per_second` is the average number of requests per seconds.
+- `compression` 
+  - `codec` (default = none): The compression method to compress the messages sent to Kafka. Four compression methods are supported: `gzip`, `snappy`, `lz4` and `zstd`.
+  - `level` (default = -1000): The corresponding compression level which measures the compression degree.
```

Comment:
any thoughts on setting the default compression level to `1` to allow users who want to configure the codec without specifying a level to do so or is it a requirement to configure both?

Reply:
Since our compression-related implementation is depended on the Sarama, I use the Sarama convention to set the default value (none and -1000). The default compression level means it doesn't use any existing compression levels.

Reply:
Right, but if a user leaves the default here `-1000`, the will trigger an error about compression level not being set if they choose a compression codec correct?

Reply:
Oh, yes, I need to fix the Validate() method.